packages/kbn-optimizer/limits.yml

```diff
@@ -10,14 +10,14 @@ pageLoadAssetSize:
   bfetch: 22837
   canvas: 29355
   cases: 180037
-  charts: 55000
-  cloud: 21076
+  charts: 92400
```

Comment:
This is a notable increase. Along with the other ones. @elastic/kibana-operations  / @gsoldevila  - are these expected through the move?